website/members/forms.py

```diff
@@ -34,4 +44,32 @@ def save(self, commit=True):
         if commit:
             member.save()
             self.save_m2m()
+
+            template = Configuration.get_mail_template("new_member")
+            if template:
+                link = reverse("admin:auth_user_change", args=(user.id,))
+                template.send(
+                    {
+                        "name": user.get_full_name(),
+                        "phone_number": member.phone_number(),
+                        "email": user.email(),
+                        "birthday": member.birthday(),
+                        "sports_card_number": member.sports_card_number(),
+                        "link_to_member": link,
+                    }
+                )
+            else:
+                mail_admins(
+                    """No template or configuration for sign up""",
+                    """Either the configuration or template for sign up is missing or 
```

Comment:
It might be nice to also say something about that there is a new member that registered